applications/StructuralMechanicsApplication/tests/test_StructuralMechanicsApplication.py

```diff
@@ -27,6 +27,7 @@
 # Simple patch tests
```

Comment:
I think you forgot `test_cook_membrane`

Reply:
:man_facepalming:  We can add it to the nightly suite.

Reply:
Added to the nightly suite.